UltraStar Play/Assets/Scenes/Main/CompareVersionException.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+
+public class CompareVersionException : Exception
+{
+    public CompareVersionException(string message) : base(message)
+    {
```

Comment:
shouldn't this contain some logging?

Reply:
nope. its just the exception class. the call base() will set the message in the base-class (like super in java). the code where the exception is catched has to do the logging - and it does.